sg_otio/sg_cut_track_writer.py

```diff
@@ -796,6 +796,20 @@ def write_shots(self, clips_by_shots, sg_project, sg_linked_entity, sg_user=None
         """
         sg_batch_data = []
         sg_shots = []
+
+        # Check if we need to find an entity cut order, to avoid making the same request for each Shot.
+        linked_entity = sg_linked_entity or sg_project
+        sfg = SGShotFieldsConfig(self._sg, linked_entity["type"])
+        entity_cut_order = None
+        if sfg.absolute_cut_order:
+            # TODO: maybe create Shot fields config before so that
+            #       we can get the field without an extra query?
+            sg_entity = self._sg.find_one(
```

Comment:
it's hard to comment without checking the whole code, but could this be done when we retrieve the `sg_linked_entity` in the first place? Seems strange to have it here... We're assuming that other fields are available in the code, so why a special case for this one?